src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -254,7 +253,8 @@ private TypeSymbol GetTypeSymbol()
             }
             else
             {
-                declType = typeBinder.BindType(_typeSyntax, diagnostics, out isVar);
+                RefKind refKind;
+                declType = typeBinder.BindType(_typeSyntax.SkipRef(out refKind), diagnostics, out isVar);
```

Comment:
This method does not produce diagnostics, so the current code is correct.

---
In reply to: [73061416](https://github.com/dotnet/roslyn/pull/12841#discussion_r73061416) [](ancestors = 73061416)